release_notes.rst

```diff
@@ -14,9 +14,116 @@ NethServer 8 releases
 
   __ http://community.nethserver.org/c/bug
 
+Major changes on 2025-04-04
+===========================
+
+**Milestone 8.4**
+
+- **Notify users of expiring passwords** -- Internal user domains with an
+  enabled expiring password policy can now be configured to send email
+  notifications to users approaching their password expiration date. See
+  :ref:`password-warning`.
+
+- **Modify external LDAP settings** -- Bind credentials and TLS settings
+  of an external user domain can now be changed after domain creation. See
+  :ref:`modify-external-ldap`.
+
+- **Set up base home directory path for applications** -- The default
+  application home directory base path is ``/home`` (per-distro default).
+  It is now possible to specify and use a different base path, as documented
+  in :ref:`disk-usage-section`, along with other techniques to expand
+  available disk space.
+
+- **Wildcard TLS certificate support** -- Wildcard certificates can be
+  uploaded and distributed to cluster applications directly from the ``TLS
+  certificates`` page. Refer to :ref:`certificate_manager-section` for
+  more information. Mail, NethVoice, and NethVoice Proxy have been updated
+  to support wildcard certificates. Support for Ejabberd is coming soon.
+
+- **New TLS-ALPN-01 default ACME challenge format** -- Let's Encrypt TLS
+  certificates are now obtained using the TLS-ALPN-01 challenge type through
+  TCP port 443. Port 80 is no longer used by new installations of
+  NethServer 8 core. Existing systems retain the previous HTTP-01 challenge
+  type and still require port 80 to be open. It is possible to upgrade to
+  the new default with the following command: ::
+
+    api-cli run module/traefik1/set-acme-server --data '{"challenge":"TLS-ALPN-01","url":"https://acme-v02.api.letsencrypt.org/directory"}'
+
+- **Traefik core module enhancements** -- The core Traefik instance,
+  running on every cluster node, has been upgraded to Traefik version 3.
+  The previous v2-compatible configuration is saved for reference under the
+  Traefik "state/" directory in ``*.v2`` directories. These can be safely
+  removed if desired.
+
+  The Traefik restoration procedure has been fixed and no longer returns a
+  conflicting Traefik instance. Instead, the backup contents are merged
+  with the active Traefik instance on the node, allowing the restoration of
+  custom HTTP routes and TLS certificates.
+
+  Additionally, this release introduces two new advanced experimental
+  features, accessible through API calls and manual configuration:
+
+  1. Support for deploying NS8 behind a network HTTP L7 Proxy.
+  2. Passing extra parameters to the Traefik container to configure custom
+     CA certificates and DNS-01 challenges.
+
+  Refer to available support channels for more information.
```

Comment:
If possible, I'd add a couple of links here

Reply:
I was thinking about some links to the dev manual but probably we still do not have such doc.
Let's keep it as is.